.github/workflows/Metrics_discovery_docker.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Build River Metrics Discovery Docker Image'
+
+on:
+    push:
+        branches:
+            - main
+        paths:
+            - 'packages/**'
+    workflow_dispatch:
+
+env:
+    SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CD_WORKFLOW_WEBHOOK_URL }}
+
+jobs:
+    build:
+        name: Build docker image
+
+        runs-on: ubuntu-latest-8-cores
```

Comment:
Does this need to run on 8-cores ?

Reply:
good question. it doesn't. using ubuntu-latest instead